### ISSUE TITLE
Combobox: add aria-roledescription

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1151]
The `aria-roledescription` attribute now has much better support. I have updated the MIND pattern to make use of it.

For example:

```html
<input aria-roledescription="Use up and down arrow keys to navigate options. Options change based on input." id="combobox-0-input" name="console0" type="text" placeholder="Playstation 4, Xbox One, etc." role="combobox" aria-expanded="true" autocomplete="list" aria-owns="combobox_0-listbox"/>
```

https://ebay.gitbook.io/mindpatterns/input/combobox

**Comment by agliga** [comment]
Trying to understand what to do with this. We already support passthrough attributes so it should be automatic. 
My understanding is we would just change examples to show the `aria-roledescription` here. Should we change all of them or only the first?

**Comment by ianmcburnie** [comment]
Sorry, I wasn't clear. Just hard code `aria-roledescription="Use up and down arrow keys to navigate options." into the template. We don't want apps to have to do this every time. 

If autocomplete is enabled (which is most of the time), the string would be slightly different: "Use up and down arrow keys to navigate options. Options change based on text input".

We can adjust the exact description text as need be, but these strings are a good place to start. Also, for I18N purposes we'll need to allow apps to override those strings with localized strings.

**Comment by ianmcburnie** [comment]
Done in 5.6.0. Closing.
